app/src/main/java/com/example/covidmodule/Application.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.covidmodule
+
+import android.app.Application
+
+class Application : Application() {
+
```

Comment:
Application is empty, this is ok? can you check this.

Reply:
Sorry about that. I forgot delete this when copy/paste code from App module. Fix it!